pkg/android/process-uploads.go

```diff
@@ -29,12 +42,8 @@ func UploadAndroidNdk(fileList []string, apiKey string, applicationId string, ve
 		err = server.ProcessFileRequest(endpoint+"/ndk-symbol", uploadOptions, fileFieldData, timeout, retries, file, dryRun)
```

Comment:
How about `ProcessFileRequest` handling this internally and not returning an `err`? It could log the message (at info level) to say it's skipped the file; and also the successful "Uploaded" message so the caller doesn't need to do this.

@lemnik – thoughts on this?

I seem to recall I've previously suggested the caller logs the success message as it has the context about what is being uploaded (e.g. "dSYM uploaded"), but actually I don't see any harm in us just using the file name like we have here for all the uploads.